### PR TITLE
grammar changes

### PR DESCRIPTION
**Pull request by Datadote**
1) your -> you
org: the vast majority of the datasets your work with
chg: the vast majority of the datasets **you** work with

2) added ,
org: When we see it in that format looks a bit boring;
chg: When we see it in that format**,** it looks a bit boring;

3) deepest -> shallowest
org: The deepest layers of our pretrained model might not need as high a learning rate as the last ones
chg: The **shallowest** layers of our pretrained model might not need as high a learning rate as the last ones

### CODE REVIEW

**Comment by sgugger** [comment]
Will just revert 3 because this is not accurate. Shallowest = last, deepest = first.
Thanks!
